winterfell/services/claimService.js

```diff
@@ -5,19 +5,11 @@ var Claim = require('./../models/claim');
 
 function ClaimService(app) {
   this.app = app;
-};
+}
 
 module.exports = ClaimService;
 module.exports.createNewClaim = function(userId, callback) {
-  return Claim.findOne({ userId: userId, state: Claim.State.INCOMPLETE }).exec(function(err, fileClaim) {
```

Comment:
I think create new claim should always do what it says, create a new claim. Checking for an existing claim and that sort of logic should be at a higher level.